cmd/pulumi.go

```diff
@@ -76,6 +82,24 @@ func NewPulumiCmd() *cobra.Command {
 				}
 			}
 
+			if rsp, err := http.Get("https://pulumi.io/latest-version"); err == nil && rsp.StatusCode == http.StatusOK {
```

Comment:
How are we ensuring that this endpoint stays up to date?

Is it better to send the current version to the service and let the service make the decision about whether its out of date instead of hardcoding that into the old CLI?

If we decide in the future that we want to allow two version to go by before warning - we won't be able to make that change in a single release with the current approach, right?

Reply:
> How are we ensuring that this endpoint stays up to date?

This is the same endpoint that `get.pulumi.com` uses today to figure out what version to install when you run it.

> Is it better to send the current version to the service and let the service make the decision about whether its out of date instead of hardcoding that into the old CLI?

Sure, we can do that instead.